frontend/public/components/storage-class-form.tsx

```diff
@@ -1022,25 +1026,27 @@ export class StorageClassForm_ extends React.Component<
 
           <div className="form-group">
             <label className="co-required" htmlFor="storage-class-reclaim-policy">
-              Reclaim Policy
+              {t('storage-class-form~Reclaim Policy')}
             </label>
             <Dropdown
-              title="Select Reclaim Policy"
+              title={`${t('storage-class-form~Select Reclaim Policy')}`}
               items={this.reclaimPolicies}
               dropDownClassName="dropdown--full-width"
               selectedKey={reclaimPolicyKey}
               onChange={(event) => this.setStorageHandler('reclaim', event)}
               id="storage-class-reclaim-policy"
             />
             <span className="help-block">
-              Determines what happens to persistent volumes when the associated persistent volume
-              claim is deleted. Defaults to &lsquo;Delete&rsquo;
+              {t(
+                'storage-class-form~storage-class-form~Determines what happens to persistent volumes when the associated persistent volume claim is deleted. Defaults to ',
+              )}{' '}
+              &lsquo;{t('storage-class-form~Delete')}&rsquo;
```

Comment:
This needs to be part of the message, not appended on later

Reply:
done